### ISSUE TITLE
The CPU architecture "arm64" is incompatible with this module

### ISSUE THREAD

**Issue by maatthc** [pact-foundation/pact-js-core#347]
I'm trying to use Arm-based AWS Graviton2 processors on EC2 but apparently **aarch64** is not supported:

`warning Resolution field "@pact-foundation/pact-node@10.12.2" is incompatible with requested version "@pact-foundation/pact-node@^10.13.8"
[2/4] Fetching packages...
error @pact-foundation/pact-node@10.12.2: The CPU architecture "arm64" is incompatible with this module.
error Found incompatible module.` 

`$ uname -mpio : 
aarch64 aarch64 aarch64 GNU/Linux
`

### Checklist

Before making a feature request, I have: 

- [x] [Searched the issues to check that this feature hasn't been requested before](https://github.com/pact-foundation/pact-js-core/issues?q=is%3Aissue)
- [x] Checked the documentation to see if it is possible to do what I want already 

### Feature description

Would be nice to have support to **aarch64** as Amazon and Oracle Cloud have good offers for instances on this platform.

### Use case

Run CI/CD pipelines on ARM processors.

**Comment by mefellows** [comment]
Might be similar to #271 #264 where it "works" if we allow it as an architecture. But we certainly don't test for it.

If you force install it does it actually work @maatthc ?

**Comment by maatthc** [comment]
The executable published by [pact-ruby-standalone](https://github.com/pact-foundation/pact-ruby-standalone) are not compatible with Linux on arm64 - so basically that needs to start there..

**Comment by TimothyJones** [comment]
That’s right. The workarounds for apple users won’t work on Linux etc. As you point out, this will become especially relevant as aarch64 becomes more popular in CI.

**Comment by mefellows** [comment]
Ah of course. We should be able to compile the new rust core for ARM.

We'll not be supporting arm for the current stable versions of Pact Node, because the standalone is unlikely to support arm anytime soon either. 

If you came here because you use Pact JS, you can try out (locally) the beta version which doesn't use Ruby 
 in the core: https://github.com/pact-foundation/pact-js/#pact-js-v3. We'll need to add arm support for that though.

**Comment by YOU54F** [comment]
pact-js-core now supports linux arm64 for both the pact ruby standalone binaries, and the pact ffi

you can find a version with support for both in 

https://www.npmjs.com/package/@pact-foundation/pact-core/v/13.15.0

**Comment by mefellows** [comment]
Amazing 🎉 !
